src/modules/infobox/utils/external-layer-providers.json

```diff
@@ -295,41 +274,40 @@
     "https://wms.geo.gr.ch/wildschutzgebiete",
     "https://wms.geo.sh.ch/wms?SERVICE=WMS",
     "https://wms.geo.ti.ch/service",
-    "https://wms.geoportal.ch:8080/geoserver/AVAI/wms",
+    "https://wms.pcn.minambiente.it/ogc?map=/ms_ogc/WMS_v1.3/Vettoriali/Rete_ferroviaria.map",
```

Comment:
is the script ordering alphabetically but takes upper cased letter as "bigger" as lower cased?
might be better to lower everything before ordering, don't you think?

Reply:
The script is ordering alphabeticaly, here is the sort compare method used; https://github.com/geoadmin/web-mapviewer/blob/37bb4cceee89bbb1aace276c863a27dda0a3e47c/scripts/check-external-layers-providers.js#L50
But that's a good idea to make the sorting non case sensitive, I'll change that

Reply:
Missed that was for the yaml files sorting, for the url sorting I use the regular `sort()` method.

Reply:
Done